Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentImportMenu.swift

```diff
@@ -204,7 +204,9 @@ struct AttachmentImportMenu: View {
 #if !targetEnvironment(macCatalyst) && !targetEnvironment(simulator)
             .onCameraCaptureModeChanged { captureMode in
                 if captureMode == .video && AVCaptureDevice.authorizationStatus(for: .audio) == .denied {
-                    microphoneAccessAlertIsVisible = true
+                    Task { @MainActor in
+                        microphoneAccessAlertIsVisible = true
+                    }
```

Comment:
Instead of these changes, annotate the action closure with `MainActor` in the `AttachmentCameraController.onCameraCaptureModeChanged(perform:)` declaration.